src/sentry/plugins/base/manager.py

```diff
@@ -117,7 +116,21 @@ def install_extensible_types(self, plugin):
         for extensible_cls in plugin.get_extensible_objects():
             self._app.extensibles.register(plugin_model, extensible_cls)
 
-    # Find
+    def get_extensible_types_from_db(self):
+        """
+        :return: class objects of extensible types
+        """
+        from clims.models.extensible import ExtensibleType
+
+        extensible_types = ExtensibleType.objects.all()
+        type_names_in_db = [e.name for e in extensible_types]
+        for type_name in type_names_in_db:
+            splitted = type_name.split('.')
```

Comment:
This should just be `split` not `splitted`.

Reply:
Ok